cmd/manager/main.go

```diff
@@ -162,10 +204,11 @@ func main() {
 			WriteTimeout: 5 * time.Minute,
 		},
 		ShutdownTimeout: &shutdownTimeout,
+		Listener:        listener,
 	}
 
 	if err := mgr.Add(&catalogServer); err != nil {
-		setupLog.Error(err, "unable to start catalog server")
+		setupLog.Error(err, "problem adding catalog server to manager")
```

Comment:
Nit: consistent error messages.

```suggestion
		setupLog.Error(err, "unable to add catalog server to manager")
```